src/components/Reports/LostTime/EditHistoryModal.jsx

```diff
@@ -403,27 +404,27 @@ const EditHistoryModal = props => {
           </>
         </Form>
       </ModalBody>
-      <ModalFooter>
-        <Button onClick={handleCancel} color="danger" style={boxStyle}>
+      <ModalFooter className={darkMode ? 'bg-yinmn-blue' : ''}>
+        <Button onClick={handleCancel} color="danger" style={darkMode ? boxStyleDark : boxStyle}>
```

Comment:
Same recommendation as above. In order to make our code more DRY, we can declare a variable like `const boxStyling = darkMode ? boxStyleDark : boxStyle;` and use it like so `style={boxStyling}